src/ranch_conns_sup.erl

```diff
@@ -43,6 +43,7 @@
 	handshake_timeout :: timeout(),
 	max_conns = undefined :: ranch:max_conns(),
 	stats_counters_ref :: counters:counters_ref(),
+	alarms = #{} :: #{term() => {undefined | reference(), map()}},
```

Comment:
This type seems wrong, see https://github.com/ninenines/ranch/pull/284/files#diff-eedbe9e800952f50286785fbf0252c1d9f5f20f003b53ca277de6e3c321e0d15R311 where there's a map in the first element of the tuple.

Reply:
I probably turned that around in the code at some point and forgot to change the type =^^=

Reply:
Dto.